css/obBase.css

```diff
@@ -1805,7 +1805,18 @@ margin: auto;
 
 .btn.btn-c3,
 #userPage .btn.btn-c3 {
```

Comment:
A color class should only have color in it. Can you make a new class and put these properties in the new class, instead of in the btn-c3 class?